python/settings.py

```diff
@@ -0,0 +1,47 @@
+class esb_settings:
+# Put your settings here:
+# IMPORTANT: Don't change the indentation!
+
+# General Settings
+    bedWidth = 220
+    bedLength = 220
+    
+    bedMargin = 5
+    filamentDiameter = 1.75
+    movementSpeed = 100
+    stabilizationTime = 20
+    bedTemp = 40
+    fanSpeed = 0
+
+    primeLength = 25
+    primeAmount = 20
+    primeSpeed = 5
+    wipeLength = 15
+    retractionDistance = 4
+    retractionSpeed = 60
+
+    blobHeight = 10
+    extrusionAmount = 200
+
+    direction = -1           #1=front to back, -1=back to front
```

Comment:
Default in the original source code is `1`, 
Suggestion, rename this field to "X-direction"  to clarify the intent of this direction field